From 41b82d75503318ed145dcda4aee77c3dd13b55c1 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 18 Jun 2024 15:45:31 +0000
Subject: [PATCH] WA -Disabling settingthe vndk and sepolicy version

This patch will disable setting the vndk and sepolicy
version to 34 as A15 has new values to it and will be
set later.

Tests Done: Build should proceed

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/compatibility_matrix.xml | 4 ++--
 groups/device-specific/caas/framework_manifest.xml   | 4 ++--
 groups/device-specific/caas/manifest.xml             | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index d941dc9..000faa8 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -39,7 +39,7 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <vendor-ndk>
+    <!--vendor-ndk>
         <version>34</version>
-    </vendor-ndk>
+    </vendor-ndk-->
 </compatibility-matrix>
diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index fec0259..58905fa 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -136,7 +136,7 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <vendor-ndk>
+    <!--vendor-ndk>
         <version>34</version>
-    </vendor-ndk>
+    </vendor-ndk-->
 </manifest>
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 6d4745f..cdd3a72 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -63,7 +63,7 @@
         </interface>
     </hal>
     <!--<kernel target-level="5"/>-->
-    <sepolicy>
+    <!--sepolicy>
         <version>34.0</version>
-    </sepolicy>
+    </sepolicy-->
 </manifest>
-- 
2.45.2

